### DISCUSSION TITLE
Additional password protection

### DISCUSSION THREAD

**Discussion by RChutchev** [arizon-dev/quickblaze-encrypt#71] [Ideas]
I propose to add an additional layer of security - password protection.
When I create a message, I want to be able to add an additional password to decrypt the message.
The people who opened that message then have to provide a password (maybe a pin) to decrypt that message.
The message should only be shown to those who provide the password.

**Comment by axtonprice** [comment]
I've transferred this discussion to an [issue](https://github.com/arizon-dev/quickblaze-encrypt/issues/72). Thanks
